Modules/Devices/test/Devices.Domain.Tests/Identities/CancelDeletionProcessTests.cs

```diff
@@ -0,0 +1,46 @@
+﻿using Backbone.BuildingBlocks.Domain;
+using Backbone.Modules.Devices.Domain.Entities.Identities;
+using Backbone.Tooling;
+using FluentAssertions;
+using Xunit;
+
+namespace Backbone.Modules.Devices.Domain.Tests.Identities;
+public class CancelDeletionProcessTests
+{
+    [Fact]
+    public void Cancel_deletion_process()
+    {
+        // Arrange
+        var currentDate = DateTime.Parse("2020-01-01");
+        SystemTime.Set(currentDate);
+
+        var identity = TestDataGenerator.CreateIdentity();
+        var device = new Device(identity);
+        identity.Devices.Add(device);
+
+        var deletionProcess = identity.StartDeletionProcessAsOwner(device.Id);
+        SystemTime.Set(DateTime.Parse("2020-01-02"));
+
+        // Act
+        identity.CancelDeletionProcess(deletionProcess.Id, device.Id);
+
+        // Assert
+        deletionProcess.Status.Should().Be(DeletionProcessStatus.Cancelled);
```

Comment:
There are soooo many assertions missing. I bet you didn't develop this in a test driven way. 😠

Reply:
Mhhh... What about assertions on
* identity.TierId
* deletionProcess.CancelledAt
* deletionProcess.CancelledBy

?

It's not hard to find out which assertions you have to make. Just look at the production code. Or you just perform TDD, as you were supposed to. Then it is impossible to forget anything.